### DISCUSSION TITLE
6770m QE/CI

### DISCUSSION THREAD

**Discussion by lfom** [chris1111/Legacy-Video-patch#18] [Q&A]
Hello!

Thanks for building the installer. You are the only person I could find that has done some work to make AMD 5xxx/6xxx work on Catalina. But do you have QE/CI enabled? I could not see any transparency in your video about the installer using a 6670 card. Many people with iMacs that have 6750/6770/6970 would benefit from this.

I do not understand why an old card like 4670 would have OpenGL enabled in Catalina but not 6xxx cards. Could you please point to some source of info about what is needed to patch the drivers to work in Catalina with 6750/6770, or perhaps what have done to make the old 4670 work?

Thanks again. Regards.

**Comment by chris1111** [comment]
I am a bit confuse on your questions because your subject speaks about a 6770m and your question relates to an ATI 4670

**Comment by lfom** [comment]
OK, I will try to break it down to make it clear.

1) How well your package work with 6670? Do you have transparency and no glitches in the search boxes?

2) Is your patch supposed to enable transparency and OpenGL for AMD 6770m?

3) If it doesn't, do you happen to know what was done so AMD 4670 work with Mojave/Catalina? Could you please point to where I can find more info about what as actually modified for it work, so perhaps I can do the same with 6750/6770?

Thanks

**Comment by chris1111** [comment]
The 6670 card works as I mentioned in the post, there is no full QE / CI for the 4670 I don't know I never use that card

**Reply by lfom** [reply]
6670 works with transparency? OpenGL is enabled for it?

**Comment by lfom** [comment]
I was able to properly enable QE/CI (at least Hackintool says it is) but still no OpenGL so there is no transparency in the UI. From what I have found, the problem is that `AMDXRadeon3000.kext` is not being loaded so the OpenGL drivers are not enabled. For some reason, the old `ATIRadeonX2000.kext` loads (that's why there is transparency/OpenGL for the 4670/4850).

Trying to load the AMDXRadeon3000 kext manually or rebuild the kext cache throws a bunch of errors about missing tables. It looks that some other file/library is missing, and I could not find any difference between High Sierra extensions and Catalina that would explain this. 

Could you please verify if your installation with AMD 6670 actually loads _AMDXRadeon3000_ kext? Thanks

**Comment by chris1111** [comment]
I have not this card insert to my optiplex now this is my Geforce 210
